src/process-mining-visualisation/mining-action.ts

```diff
@@ -10,18 +10,18 @@ export interface MiningAction extends Action {
 }
 
 export interface MiningData {
-  processname: string;
-  analysistype: string;
-  numberofinstances: number;
+  processName: string;
+  analysisType: string;
+  numberOfInstances: number;
   nodes: MiningNode[];
-  timeframe: period;
+  timeFrame: period;
 }
 
 export interface MiningNode {
   type: string;
-  ID: string;
-  relativevalue: number;
-  labelvalue: string;
+  id: string;
+  relativeValue: number;
+  labelValue: number;
```

Comment:
We change this type due to the requirement from story: https://1ivy.atlassian.net/browse/MARP-974

Reply:
Looks fine to me :smile: